ajax/libs/react-uwp/package.json

```diff
@@ -28,7 +28,7 @@
       {
         "basePath": "",
         "files": [
-          "**/*.+(js|css|svg|ttf|woff|woff2|eot|png|jpg|jpeg)"
+          "!(LICENSE)"
```

Comment:
Sorry that I didn't say it clear enough, the whole line should be `"**/!(LICENSE)"`